newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -229,7 +252,11 @@ const BuildSection = ({
           ? 3
           : 5,
         numberOfItemsInCarousel: showAllGameTemplates ? 0 : isMobile ? 8 : 12,
-        numberOfItemsInGrid: showAllGameTemplates ? 60 : isMobile ? 16 : 20,
+        numberOfItemsInGrid: showAllGameTemplates
+          ? allGameTemplatesCount + pageIndex * pageSize
```

Comment:
I'm not sure I understand this part? Maybe the naming is confusing?
How can we display all the game templates PLUS some more pages ?

Reply:
Yes, the naming is not great. I changed it for something less concise but clearer